apps/web/src/pages/ApplicantDashboardPage/components/CareerDevelopmentTaskCard.tsx

```diff
@@ -0,0 +1,276 @@
+            defaultMessage: "Career planning",
+            id: "zN7MBv",
+            description: "Title for a users career plan",
+          })}
+        </>
+      ),
+    },
+  ];
+
+  return (
+    <>
+      <div
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1)"
+        data-h2-flex-grow="p-tablet(2)"
+      >
+        <TaskCard.Root
+          icon={Cog8ToothIcon}
+          title={intl.formatMessage({
+            defaultMessage: "Career development",
+            id: "+2/ksA",
+            description: "Card title for career development",
+          })}
+          headingColor="secondary"
+          headingAs="h2"
+        >
+          <TaskCard.Item>
+            <Accordion.Root
+              type="multiple"
+              data-h2-padding-bottom="base:selectors[>.Accordion__Item > .Accordion__Content](x.5)"
+            >
+              <Accordion.Item value="your_career_planning">
+                <Accordion.Trigger
+                  as="h3"
+                  subtitle={intl.formatMessage({
+                    defaultMessage:
+                      "Opt in to your profile information being shared with recruiters and talent management staff so that you can be referred for job opportunities or training.",
+                    id: "L4UWvU",
+                    description:
+                      "Subtitle explaining career planning expandable withing career development card",
+                  })}
+                >
+                  {intl.formatMessage({
+                    defaultMessage: "Career planning",
+                    id: "E0I0Yj",
+                    description: "Career planning expandable",
+                  })}
+                </Accordion.Trigger>
+                <Accordion.MetaData metadata={careerPlanningMetaData} />
+                <Accordion.Content>
+                  <div
+                    data-h2-display="base(flex)"
+                    data-h2-flex-direction="base(column)"
+                    data-h2-gap="base(x1)"
+                    data-h2-padding-top="base(x.5)"
+                  >
+                    <div>
+                      <p data-h2-font-weight="base(700)">
+                        {intl.formatMessage({
+                          defaultMessage:
+                            "Interest in promotions and lateral moves",
+                          id: "TiPh1V",
+                          description:
+                            "Heading for list of user's interest in employment moves",
+                        })}
+                      </p>
+                      {moveInterests ? (
+                        <MoveInterestsList
+                          moveInterests={moveInterests}
+                        ></MoveInterestsList>
+                      ) : (
+                        <p>
+                          {intl.formatMessage({
+                            defaultMessage: "<red>Missing information</red>",
```

Comment:
Consider removing the <red> formatting from the string and doing it on the `<p>` element instead so the string can be reused.

Reply:
Done in https://github.com/GCTC-NTGC/gc-digital-talent/commit/9368ed0dab4b494a18a55b8fefc2ac88e899f0b8